system/core/events.php

```diff
@@ -74,7 +162,7 @@ function hypha(url, cmd, arg) {
 		document.getElementById('argument').value = arg;
 		document.forms['hyphaForm'].action = url;
 		for(i=0; i<postProcessingList.length; i++) postProcessingList[i]();
-		if (cmd||arg) $(document.forms['hyphaForm']).submit();
+		if (cmd||arg) (document.forms['hyphaForm']).submit();
```

Comment:
What is this change for? I believe this switches from the jquery version of `submit()` to the regular version, which IIRC are not completely identical. If this is indeed needed for the info icons, the commit message should probably explain why, and this could perhaps also be moved into a separate commit.

Reply:
In my context the $(comment) triggers an error.
not reproducable, so added the $ again.